python/rikai/spark/sql/codegen/base.py

```diff
@@ -67,28 +98,19 @@ def udf_from_spec(spec: ModelSpec):
             f"Only spec version 1.0 is supported, got {spec.version}"
         )
 
-    if spec.flavor in KNOWN_FLAVORS:
-        codegen_module = f"rikai.spark.sql.codegen.{spec.flavor}"
-    else:
-        codegen_module = f"rikai.contrib.{spec.flavor}.codegen"
-
     schema = spec.model_type.dataType()
 
     @udf(returnType=schema)
     def deserialize_return(data: bytes):
         return _pickler.loads(data)
 
-    try:
-        codegen = importlib.import_module(codegen_module)
-        return (
-            pickle_udt,
-            codegen.generate_udf(spec),
-            deserialize_return,
-            schema,
-        )
-    except ModuleNotFoundError:
```

Comment:
is it a good idea to just let random model flavors in?

Reply:
no changes for this part, these deleted code is refactored in `codegen_from_spec` and is reused in `func_from_spec`